### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by MpiloMshengu**
refractor

### CODE REVIEW

**Review by CastelloDev — DISMISSED** [review]

File: src/functions.js [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
+    return fs.readFileSync(fileDirectory);
+}
+
+export const exportSvg = (svgo, data, fileName) => {
+    svgo.optimize(data)
+    .then(result => {
+        let exportDirectory = EXPORT_DIRECTORY + fileName;
+
+        tools.ExportSVG(result.data, exportDirectory)
+        .then(() => console.log('Exported: '+fileName))
```
Comment (hvandermerwe) [inline src/functions.js]: missing semicolon

File: src/functions.js [hunk 2]
```diff
@@ -1,14 +1,51 @@
...
+        .catch(err => console.log(err));
+    })
+    .catch(err => console.log(err));
+}
+
+export const buildReport = directory => {
+    console.log('\ndirectory: '+directory);
```
Comment (hvandermerwe) [inline src/functions.js]: you can rather make use of string interpolation

File: src/functions.js [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+    const fileNames = getAllFileNames(directory);
+    fileNames.forEach(fileName=> {
+        const filePath = directory + fileName;
+        const stat = fs.statSync(filePath)
+        console.log(fileName + '' + stat.size)
+    })
+}
+
+export const comparisonReport = (origonalDirectory, optimizedDirectory) => {
```
Comment (hvandermerwe) [inline src/functions.js]: typo

File: src/constants.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export const IMPORT_DIRECTORY = './src/svgs/';
```
Comment (hvandermerwe) [inline src/constants.js:1]: I would suggest using path.join() to set up your directories

**Review by hvandermerwe — CHANGES_REQUESTED** [review]
please fix

**Review by hvandermerwe — DISMISSED** [review]

**Review by CastelloDev — APPROVED** [review]

**Review by hvandermerwe — APPROVED** [review]

**Pull request merged**